site/docs/education/egeria-dojo/developer/different-types-of-apis-platform-deployment.md

```diff
@@ -103,7 +103,7 @@ In `egeria-dev-projects`, there is a module called `egeria-platform-report`.  Th
               Asset Owner OMAS: Manage an asset
               Stewardship Action OMAS: Manage exceptions and actions from open governance
               Governance Program OMAS: Manage the governance program
-              Digital Service OMAS: Manage a digital service's lifecycle
+              Digital Service OMAS: Manage a digital service through its lifecycle
```

Comment:
it's ?

Reply:
The apostophe is a problem to the formatting